frontend/src/metabase/visualizations/visualizations/SmartScalar/compute.js

```diff
@@ -1,16 +1,20 @@
 // eslint-disable-next-line no-restricted-imports -- deprecated usage
 import moment from "moment";
+import _ from "underscore";
 import { t } from "ttag";
 import * as Lib from "metabase-lib";
-import { formatValue } from "metabase/lib/formatting/value";
 import { formatDateTimeRangeWithUnit } from "metabase/lib/formatting/date";
-import { color, colors } from "metabase/lib/colors";
 import { COMPARISON_TYPES } from "metabase/visualizations/visualizations/SmartScalar/constants";
 import { formatChange } from "metabase/visualizations/visualizations/SmartScalar/utils";
 import { isEmpty } from "metabase/lib/validate";
 import { isDate } from "metabase-lib/types/utils/isa";
 
-export function computeTrend(series, insights, settings) {
+export function computeTrend(
+  series,
+  insights,
+  settings,
+  { formatValue, color },
```

Comment:
Really nit, but we use `getColor` everywhere when we inject the function. It makes it more obvious that it is not our imported `color` function and with verb prefix it looks like a bit more precise name

Reply:
Fixed in [`441c424`](https://github.com/metabase/metabase/pull/37602/commits/441c4244deb93ce005de0981dc13bd607e201493)